client/modules/workspace/src/JobsDetailModal/JobsDetailModal.tsx

```diff
@@ -298,7 +296,7 @@ export const JobsDetailModal: React.FC<{ uuid: string }> = ({ uuid }) => {
       className={`${styles.root} job-history-modal`}
       title={
         <header>
-          Job Detail: {uuid}
+          Job Detail: {jobData.name}
```

Comment:
```suggestion
          Job Detail: {jobData?.name}
```
Locally, this comes back as `undefined` for me without the optional chaining.

Reply:
Thanks, fixed